src/app/themes.tsx

```diff
@@ -4,58 +4,151 @@ import GlobalStyle from "@/app/global_styles";
+    light: true,
+    main: {
+        background: ["#fdfdfd", "#f6f6f6", "#eeeeee", "#d0d0d0"],
+        foreground: ["#2d2d2d", "#2d2d2d", "#2d2d2d", BLACK],
+        largeForeground: ["#2d2d2d", "#2d2d2d", "#2d2d2d", BLACK],
+        lighterForeground: ["#666666", "#666666", "#666666", "#3b3b3b"],
+        border: "#d5d5d5",
+    },
+    primary: {
+        background: ["#e7f5e9", "#7fd495", "#02985a", "#05663f", "#222f28"],
+        foreground: [BLACK, BLACK, BLACK, WHITE, WHITE],
+        largeForeground: [BLACK, BLACK, WHITE, WHITE, WHITE],
+    },
+    accent: {
+        background: "#1b385f",
+        foreground: WHITE,
+        largeForeground: WHITE,
+    },
+    error: "#ff624e",
+    shadow: "#e2e2e2",
+    rainbow: {
+        lightRed: {
+            background: "#fa9189",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkRed: {
+            background: "#fc4942",
+            foreground: BLACK,
+            largeForeground: WHITE,
+        },
+        lightOrange: {
+            background: "#fcae7c",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkOrange: {
+            background: "#fd7e2a",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightYellow: {
+            background: "#ffd868",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkYellow: {
+            background: "#ffbb00",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightGreen: {
+            background: "#b4f5b3",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkGreen: {
+            background: "#5fa881",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightBlue: {
+            background: "#acd5df",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkBlue: {
+            background: "#7bbafc",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightPurple: {
+            background: "#ceaae9",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkPurple: {
+            background: "#b18ff9",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        lightGrey: {
+            background: "#d7d7d7",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkGrey: {
+            background: "#808080",
+            foreground: BLACK,
+            largeForeground: WHITE,
+        },
+        lightBrown: {
+            background: "#bfaba2",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+        darkBrown: {
+            background: "#a68f68",
+            foreground: BLACK,
+            largeForeground: BLACK,
+        },
+    },
```

Comment:
Is it worth choosing darker colours for `dark*` that would be dark enough for a white foreground colour so it has more contrast with the lighter colours?

Reply:
To be fair it is mainly used in calendar
And in calendar it is better for text to have black color otherwise the week and day view will look inconsistent with different text color